CONTRIBUTING.md

```diff
@@ -44,6 +44,11 @@ Some useful commands include:
   pre-commit-hooks.nix
 - `just itest`: runs integration tests
 
+For non-nix users,
+There are traditional `pre-commit` hooks, which you can install with your system package manager or
+`brew|pip install pre-commit`, and run `pre-commit install -c .pre-commit-config.yml` in the root of the repository.
```

Comment:
Because it already took me a minute to realize that it's now `.yml` and not `.yaml`

Reply:
haha here i thought i was being clever

Reply:
To be fair it was kinda clever, but yes, it's probably smart to be more verbose if we're not doing the default anyways.

Reply:
To be honest, I do actually like the clever approach ... makes non-nix devs feel less excluded ... but I'm afraid this could come and haunt us at some point ... people mixin those two files up, or thinking it's a typo ... so probably better to stick with explicit.